iOS/SkyService/Data/DataService/DataService+Prepopulate.swift

```diff
@@ -323,11 +323,11 @@ extension DataService {
                     "workspaceId": workspaceId,
                     "isCrewOnly": isCrewOnly,
                     "deleted": false
-                ], writeStrategy: .insertIfAbsent)
+                ])
             }
 
             for menuItem in menuItemsData {
-                let _ = try? tx["menuItems"].upsert(menuItem, writeStrategy: .insertIfAbsent)
```

Comment:
Should the pre-populated menu items use `insertDefaultWithAbsent`? I'm thinking this might be important because multiple people could prepopulate the app with the same menu items concurrently which would create a lot of extra unintentional metadata.

Reply:
You're correct - I changed it. thanks!
- [348f360](https://github.com/getditto/demoapp-skyservice/pull/19/commits/348f36099bf824b993cea00fdf05c15e827e9c59)